distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -675,21 +675,42 @@ type Column
         run_vectorized_binary_op self "like" (_ -> _ -> Error.throw (Illegal_State_Error "The `Like` operation should only be used on Text columns.")) pattern
 
     ## Checks for each element of the column if it is contained within the
-       provided vector.
+       provided vector or column.
 
        Arguments:
-       - vector: A vector of elements. The resulting column will contain true at
-         the positions where the corresponding element of `self` is contained
-         in `vector`.
+       - vector: A vector of elements or another column. The resulting column
+         will contain true at the positions where the corresponding element of
+         `self` is contained in `vector`.
 
        > Example
          Check if elements of a column are contained in a provided vector.
 
              import Standard.Examples
 
              example_contains = Examples.text_column_1.is_in [1, 2, 5]
+    is_in : Column | Vector -> Column
     is_in self vector =
-        run_vectorized_binary_op self "is_in" (elem -> vector -> vector.contains elem) vector skip_nulls=False
+        op_name = "is_in"
+        result_name = "Result"
```

Comment:
lets pass this in as an argument - seems odd to just have the constant here.

Reply:
Maybe default to `IN_<exsiting_name>`

Reply:
This will be handled as part of https://www.pivotaltracker.com/story/show/183615456